python/lsst/ts/fbs/utils/maintel/surveys.py

```diff
@@ -98,15 +98,17 @@ def generate_image_survey(
         gap_min=target.visit_gap,
     )
 
-    sequence = [empty_observation() for i in range(len(target.filters))]
+    sequence = [ObservationArray(n=1) for i in range(len(target.filters))]
 
     for filter_obs, observation in zip(target.filters, sequence):
         observation["RA"] = target.ra.to(u.rad).value
         observation["dec"] = target.dec.to(u.rad).value
         observation["filter"] = filter_obs
         observation["exptime"] = target.exptime
         observation["nexp"] = target.nexp
-        observation["note"] = f"{target.survey_name}:{target.target_name}"
+        observation["scheduler_note"] = f"{target.survey_name}:{target.target_name}"
+        observation["target_name"] = target.target_name
+        observation["science_program"] = target.survey_name
```

Comment:
This is not a bug, but I'm surprise we settled on survey_name for the "science_program" in the observation array. I though we were trying to unify the usage of "science_program" across all metadata interfaces so I kind of expected something like the BLOCK-ID here...maybe I'm misremembering though.